apps/src/code-studio/pd/workshop_enrollment/enroll_form.tsx

```diff
@@ -430,23 +491,24 @@ function EnrollForm(props) {
     'Unsure / Still deciding',
   ];
 
-  const csfCourses = Object.keys(CSF_COURSES)
-    .filter(key => key !== 'courses14_accelerated')
-    .map(key => CSF_COURSES[key])
-    .concat([
-      {
-        answerText: `${OTHER} ${EXPLAIN}`,
-        inputValue: formState.explain_csf_course_other,
-        onInputChange: handleCsfCourseOtherChange,
-      },
-    ]);
+  const csfCourses = [
+    ...Object.values(CSF_COURSES),
+    {
+      answerText: `${OTHER} ${EXPLAIN}`,
+      inputValue: formState.explain_csf_course_other,
+      onInputChange: handleCsfCourseOtherChange,
+    },
+  ];
   const previousCourses = props.previous_courses.concat([
     "I don't have experience teaching any of these courses",
   ]);
 
   const roles =
-    (props.workshop_course === CSF && CSF_ROLES) ||
-    (props.workshop_course === ADMIN_COUNSELOR && ADMIN_COUNSELOR_ROLES);
+    props.workshop_course === CSF
+      ? CSF_ROLES
+      : props.workshop_course === ADMIN_COUNSELOR
+      ? ADMIN_COUNSELOR_ROLES
+      : [];
```

Comment:
while the `workshop_course` is probably either `CSF` or `ADMIN_COUNSELOR`, if it happened to be neither than roles would be `false` instead of an array, which could trigger a type error

Reply:
This is more of a personal nit so please feel free to disregard: I find the `x || y || default` easier to read than the nested terniary `condition ? x : condition ? y : default`, would it be possible to just append to the line `|| []`?

Reply:
yeah, I agree a nested ternary is gross to look at...

I'm not sure we can do `|| []` because we need to check the value of `workshop_course` to determine what the array should contain. `x || y || default` only gets to the default if x and y are both falsey values, and `workshop_course` could be undefined or a value other than `CSF` or `ADMIN_COUNSELOR`

we could do a helper method with a switch case, that would be much easier to read I think

```
getRoles(workshop_course) {
  switch (workshop_course) {
    case CSF:
      return CSF_ROLES;
    case ADMIN_COUNSELOR:
      return ADMIN_COUNSELOR_ROLES;
    default:
      return []
  }
}
```

Reply:
or... we could just define a `WORKSHOP_ROLES` object that maps the course name to an array of roles 🤔 

```
const WORKSHOP_ROLES = {
  CSF: CSF_ROLES,
  ADMIN_COUNSELOR: ADMIN_COUNSELOR_ROLES
}

const roles = WORKSHOP_ROLES[props.workshop_course] || []
```

Reply:
lol, typescript says no 🚫 

![Screenshot 2024-10-24 at 10 16 04 AM](https://github.com/user-attachments/assets/14c24247-e4ba-4ffa-91c8-a05b5b6c1c4e)

Reply:
Ohhh gotcha gotcha, then I'm fine with either the helper function or keeping it how you had it!